ql/src/java/org/apache/hadoop/hive/ql/stats/BasicStatsTask.java

```diff
@@ -118,11 +118,15 @@ public String getName() {
     private boolean isMissingAcidState = false;
     private BasicStatsWork work;
     private boolean followedColStats1;
+    private boolean isBasicStatProvided;
+    private Map<String, String> providedBasicStats;
 
-    public BasicStatsProcessor(Partish partish, BasicStatsWork work, HiveConf conf, boolean followedColStats2) {
+    public BasicStatsProcessor(Table table, Partish partish, BasicStatsWork work, boolean followedColStats2) {
```

Comment:
My feeling is that `Partish` should be some general object covering both `Table` and `Partition`. How hard/wasteful would it be to add the required data to the `Partish` object?

Reply:
yes; you should use the partish: `Partish.buildFor(table)`

adding a `Table` here will cause confusion...

Reply:
Removed it.